### PR TITLE
docs updates

### PR DESCRIPTION
**Pull request by birdsarah**
- Update header and footer links in docs site
- Add migration note about js_resources, css_resources
- Update file_html docstring

### CODE REVIEW

**Comment by birdsarah** [comment]
one more update coming on this

**Comment by birdsarah** [comment]
@mattpap, at the least, can you double check my updated docstring for file_html

File: bokeh/embed.py [hunk 1]
```diff
@@ -286,8 +286,8 @@ def file_html(models,
     Args:
         models (Model or Document or list) : Bokeh object or objects to render
             typically a Model or Document
-        resources (Resources) : a resource configuration for Bokeh JS & CSS assets. Pass ``None`` if
-            using js_resources of css_resources manually.
+        resources (Resources) : a resource configuration for Bokeh JS & CSS assets. To use
+            js_resources or css_resources manually, pass a tuple of (js_resources, css_resources).
```
Comment (mattpap) [inline bokeh/embed.py]: I would say "and/or" and signify that `None` is an accepted value. Also `(js_res, css_res)` should be in double backticks.

File: bokeh/embed.py [hunk 2]
```diff
@@ -286,8 +286,8 @@ def file_html(models,
     Args:
         models (Model or Document or list) : Bokeh object or objects to render
             typically a Model or Document
-        resources (Resources) : a resource configuration for Bokeh JS & CSS assets. Pass ``None`` if
-            using js_resources of css_resources manually.
+        resources (Resources) : a resource configuration for Bokeh JS & CSS assets. To use
```
Comment (mattpap) [inline bokeh/embed.py]: Or maybe the type should say it all, i.e. `Resources or tuple(JSResources or None, CSSResources or None)`.

File: sphinx/source/docs/releases/0.11.0.rst [hunk 3]
```diff
@@ -91,5 +91,17 @@ Old location               New location
...
 
+Before: 
 
+  ``html = file_html(layout, None, title=title, template=template, js_resources=js_resources, css_resources=css_resources)``
+
+v0.11:
+
+  ``html = file_html(layout, resources=(js_resources, css_resources), title=title, template=template)``
+
+Note you can pass ``None`` in place of either ``js_resources`` or ``css_resources``.
```
Comment (mattpap) [inline sphinx/source/docs/releases/0.11.0.rst]: I was certain you added `{js,css}_resources` in between 0.10 and 0.11. Maybe this was a last minute addition for 0.10?

**Comment by birdsarah** [comment]
> I was certain you added {js,css}_resources in between 0.10 and 0.11. Maybe this was a last minute addition for 0.10?

oh, poop - checking.

File: bokeh/embed.py (hunk 2 above)
Comment (birdsarah) [inline bokeh/embed.py]: will update

**Comment by birdsarah** [comment]
checked - it was in 0.10.0 (https://github.com/bokeh/bokeh/blob/0.10.0/examples/interactions/interactive_bubble/gapminder.py#L126)

**Comment by damianavila** [comment]
LGTM, merging now...

**Pull request merged**